packages/mobile/locales/en-US/global.json

```diff
@@ -116,5 +118,12 @@
   "message": "Message",
   "addressLookupFailure": "Failed to look up phone number address",
   "saltFetchFailure": "Failed to retrieve phone number secret",
-  "saltQuotaExceededError": "Your phone number lookup quota is used up"
+  "saltQuotaExceededError": "Your phone number lookup quota is used up",
+  "addressValidationNoMatch": "The account information entered does not match any associated with your contact",
```

Comment:
The issue I ran into is that `showError` [forces use of the global namespace](https://github.com/celo-org/celo-monorepo/blob/master/packages/mobile/src/alert/actions.ts#L40). I can either leave as is or switch to using `showAlert` and move everything from `global` into `sendFlow7`. What do you think?

As an FYI - I don't think past decisions should necessarily dictate the future but looks like there are quite a few other instances of flow-specific error messages included in `global`

Reply:
Hmm, it shouldn't be forcing global. If you check ErrorMessage.ts you'll see that many include a namespace, which we should be doing more of going forward.
I see that line you linked to was changed only 3 days ago. Maybe @etuleu's recent commit broke the i18n namespacing for error. @etuleu can you check this please?

Reply:
Checked the commit history and looks like this has been an issue since initial commit? I first noticed this ~3 weeks ago and mentioned it in that large thread of observations I posted to the wallet channel.

I don't know much about`i18n`... does it have built in functionality that parses namespaces? If the fix isn't a quick one, I am more inclined to address in a separate issue so we can get this merged.

Reply:
Yep totally reasonable